build.sh

```diff
@@ -224,6 +224,17 @@ BUILD_CUSTOM_TARGETS=
 UNAME=$(uname)
 LC_UNAME=$(echo "${UNAME}" | tr '[:upper:]' '[:lower:]')
 
+if [[ "$LC_UNAME" == "linux" ]]; then
+    # On Linux both GCC and Clang are commonly available, but CMake
```

Comment:
maybe exiting when clang is not available would be helpful as well.  Otherwise, we'd get a cmake error that's less scrutable (?).

Reply:
actually, looks like this will break our github workflows since we're not using the latest clang version on system but a fixed one.

Reply:
I'm still struggling to build anyway.    E.g. the SDL2 build isn't picking up stdint.h typedefs, and I am not chasing everything down.  So this is getting opportunistic for me.